styleguide/source/_patterns/01-atoms/button.md

```diff
@@ -0,0 +1,28 @@
+### Description
+This is the standard button pattern
+
+### Variant options
+* Using a 
+* [link](./?p=atoms-button-as-link) instead of a button
+* [small](./?p=atoms-button-as-small)
+* [solid](./?p=atoms-button-as-outline)
+* [outline](./?p=atoms-button-as-outline)
```

Comment:
While I like that we're noting variants in the markdown file, I find it confusing that these do not currently exist. I would recommend that we refrain from adding variant information until we are adding in the variants.